README.md

```diff
@@ -4468,18 +4468,35 @@ More on this topic [here](https://kubernetes.io/docs/concepts/services-networkin
 </b></details>
 
 <details>
-<summary>What services types are there?</summary><br><b>
+<summary> What does being cloud-native mean?</summary><br><br>
 </b></details>
 
 <details>
-<summary>Explain Liveness probe</summary><br><b>
```

Comment:
Why did you remove this question?

Reply:
It was a mistake, I think when I forked to my local repo it had issues